VTUL/dlp-access
# New backend with correct field mappings in search indices.

**New backend with correct field mappings in search indices.**
* * *

**JIRA Ticket**: (link) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
New backend with correct field mappings in search indices.

# What's the changes? (:star:)
A in-depth description of the changes made by this PR. Technical details and possible side effects.

* Adds dlpdev backend with more complete search indices

# How should this be tested?
* Pull dlpdev application branch
* Checkout dlpdev backend environment
* Run tests
* Do visual check to see that the application is working as expected

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

Example:
* application branch: `dlpdev`
* backend env: `dlpdev`

# Interested parties
@yinlinchen 

(:star:) Required fields
